Documentation/guides/messages/xa0111.md

```diff
@@ -17,14 +17,17 @@ aapt2 -version
 
 ## Solution
 
-Try re-installing the android build-tools.
+Check for a custom `<Aapt2ToolPath>` value in your _.csproj_ file and
+remove it, as Xamarin.Android redistributes a valid copy of AAPT2.
+Alternatively, try re-installing the Xamarin workload in Visual
+Studio.
 
-`aapt` is part of the Android SDK and is used internally by
+AAPT is part of the Android SDK and is used internally by
```

Comment:
It occurs to me that one "problem" with ``s/`aapt`/AAPT/g`` is that maybe, *maybe*, Acrolinx will have problems with it, and "ding" us for it. See e.g. https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9d312bb2-4a2c-4646-9671-00ab2510881b (via https://github.com/MicrosoftDocs/xamarin-docs-pr/pull/1833), wherein Acrolinx "complains" that the following words are misspelled:

  * binutils
  * dex
  * multi-dex
  * dex-compiler
  * aapt
  * abi
  * Iso (via `Hebew (Iso alternative)`)

In local testing, all-caps *appears* to be skipped by Acrolinx, which is good.  Using backpacks `` `aapt` `` also appears to appease Acrolinx, but bold `**aapt**` does *not* appease it.

Reply:
I'll vote for "AAPT" when not referring specifically to the command line command. The motivation is the distinction between AAPT2 and `aapt2` in the introductory section of <https://developer.android.com/studio/command-line/aapt2>.